### ISSUE TITLE
SRT `Application/x-subrip` Transcripts Giving "transcript format is not supported" Error

### ISSUE THREAD

**Issue by leoherzog** [Automattic/pocket-casts-android#2912]
### Description

My podcast publishes episodes with transcripts ([spitball.show](https://spitball.show/)).

```xml
<item>
    <title>YouTuber Stock Market, Pit Crew Medical Combos, Smash Trash For Cash, and Marketing Team Game Shows</title>
    <enclosure url="https://op3.dev/e,pg=65fcb3c4-f35f-562a-b7bb-4513fd71a35d/spitball.show/audio/@podcast/25.mp3" length="57097510" type="audio/mpeg"/>
    <guid>https://spitball.show/@podcast/episodes/25</guid>
    <pubDate>Mon, 09 Sep 2024 10:00:00 +0000</pubDate>
    <description>
        <![CDATA[<p>Special thanks to Greg for joining us on this episode!</p><p>00:00:00 - Intro<br />
00:00:42 - Nostalgic Noises<br />
00:06:58 - YouTuber Stock Market<br />
00:20:42 - Bonus Idea: Tuition Investing<br />
00:24:55 - Pit Crew Medical Combos<br />
00:32:34 - Smash Trash for Cash<br />
00:43:39 - Marketing Team Game Shows<br />
00:54:02 - Outro</p><footer><p>We’d love to hear from you!<br />
📧 <a href="mailto:podcast@spitball.show">podcast@spitball.show</a><br />
⁂ Fediverse — @<a href="mailto:podcast@spitball.show">podcast@spitball.show</a></p><p>🌐 <a href="https://spitball.show">https://spitball.show</a><br />
🔔 <a href="https://spitball.show/subscribe">https://spitball.show/subscribe</a><br />
👽 <a href="https://spitball.show/reddit">https://spitball.show/reddit</a><br />
🎥 <a href="https://spitball.show/youtube">https://spitball.show/youtube</a></p><p>🎶 Our intro/outro music is Swingers by the Bonkers Beat Club.</p></footer>]]>
    </description>
    <itunes:duration>3300</itunes:duration>
    <link>https://spitball.show/@podcast/episodes/25</link>
    <itunes:image href="https://s3.fr-par.scw.cloud/s3.castopod.cloud/c89b2cfc0e4c784122216b2ad9d5d54b7/podcasts/podcast/cover_feed.png"/>
    <itunes:episode>25</itunes:episode>
    <itunes:episodeType>full</itunes:episodeType>
    <podcast:episode>25</podcast:episode>
    <podcast:comments uri="https://spitball.show/@podcast/episodes/25/comments" contentType="application/podcast-activity+json"/>
    <podcast:socialInteract uri="https://spitball.show/@podcast/posts/7f95e78e-dbc9-456d-b07b-6434bda936a7" priority="1" platform="castopod" protocol="activitypub" accountId="@podcast@spitball.show" pubDate="2024-09-09T10:00:00+0000"/>
    <podcast:transcript url="https://s3.fr-par.scw.cloud/s3.castopod.cloud/c89b2cfc0e4c784122216b2ad9d5d54b7/podcasts/podcast/25-transcript.srt" type="application/x-subrip" rel="captions" language="en"/>
    <podcast:chapters url="https://s3.fr-par.scw.cloud/s3.castopod.cloud/c89b2cfc0e4c784122216b2ad9d5d54b7/podcasts/podcast/25-chapters.json" type="application/json+chapters"/>
</item>
```

Other podcast players such as Apple Podcasts and [steno.fm](https://www.steno.fm/show/65fcb3c4-f35f-562a-b7bb-4513fd71a35d/episode/aHR0cHM6Ly9zcGl0YmFsbC5zaG93L0Bwb2RjYXN0L2VwaXNvZGVzLzI1) are pulling in the transcripts properly. However, on Pocket Casts, I get the error:

`Sorry, but this transcript format is not supported: application/x-subrip`

### Step-by-step reproduction instructions

1. Begin to play "Spitball — Startup Ideas on Tap"
https://pca.st/podcast/e1f3ae20-fa29-013b-f549-0acc26574db2
2. Press the 🗨️ Transcripts button on the toolbar

### Screenshots or screen recording

![Sorry, but this transcript format is not supported: application/x-subrip](https://github.com/user-attachments/assets/dcaa8159-593d-4da5-9280-def7d4831454)

### Did you search for existing bug reports?

- [X] I have searched for existing bug reports.

### Device, Operating system, and Pocket Casts app version

Google Pixel 8, Android 14, Pocket Casts 7.73-rc-1

**Comment by leoherzog** [comment]
Also occurring on iOS, Issue [#2185](https://github.com/Automattic/pocket-casts-ios/issues/2185)

**Comment by MiSikora** [comment]
Fixed in #2926
